### PR TITLE
fix: substrafl CI

### PR DESCRIPTION
**Pull request by louishulot**
We decided to use the latest substra-tools image for the test starting from now as we already use it for:
- the benchmark 
- the metric 

Substratools is now installed from the répo and not pypi

Some small updates so the test passes

### CODE REVIEW

**Review by Fabien-GELUS — APPROVED** [review]
LGTM

File: substrafl/remote/register/generate_wheel.py [hunk 1]
```diff
@@ -138,7 +138,7 @@ def pypi_lib_wheels(lib_modules: List, operation_dir: Path, python_major_minor:
                 )
             except subprocess.CalledProcessError as e:
                 raise ConnectionError(
-                    "Couldn't access to Owkin pypi, please ensure you have access to https://pypi.owkin.com/simple/.",
+                    "Couldn't access to pypi, please ensure you have access to https://pypi.org",
```
Comment (Esadruhn) [inline substrafl/remote/register/generate_wheel.py:141]: maybe we can remove the try except now, no need to try catch to say to the user they don't have access to pypi

File: tests/remote/register/test_generate_package.py [hunk 2]
```diff
@@ -43,7 +43,7 @@ def test_generate_pypi_lib_wheel(session_dir):
 
     # save the current versions the libs to set them back later
     substratools_version = substratools.__version__
-    substratools.__version__ = "0.9.1"
+    substratools.__version__ = "0.7.0"
```
Comment (Esadruhn) [inline tests/remote/register/test_generate_package.py:46]: why downgrade this?

File: substrafl/remote/register/generate_wheel.py (hunk 1 above)
Comment (louishulot) [reply substrafl/remote/register/generate_wheel.py:141]: yup

**Comment by louishulot** [comment]
slow test are also pasing (just reverted the commit)

File: tests/remote/register/test_generate_package.py (hunk 2 above)
Comment (Esadruhn) [reply tests/remote/register/test_generate_package.py:46]: ah and since you only generate the wheel it is OK!

**Pull request merged**